tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+- SAR backscattering depends on the **angle of the incident microwave radiation**. Thus, since the side-looking SAR operates over a range of incidence angles along the swath, the same target will appear different whether it is in near range (low incidence angle) or far range (higher incidence angle) of the scene. The manner in which the backscattering varies with incidence angle depends on the target: a flat dry soil surface has a stronger drop off with incidence angle than, for instance, a forest.
+- **Terrain relief** has a strong effect on SAR backscattering because it modulates the area that is illuminated by the side-looking SAR radiation. Slope angle determines the orientation with respect to the incident radiation. This causes foreshortening for slopes oriented towards the SAR and shadowing of slopes steeper than the local incidence and directed away from the SAR.
+
+In order to use SAR data efficiently, you need to be aware of what part of the signal variation relates to the intrinsic properties of the sensor, and what part is linked to the physical properties of the area you are looking at. 
+
+## Sentinel-1
+
+Let's look at some of the issues outlined above and how these relate to practical use of [Sentinel-1](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/overview). Sentinel sensor data is generated by the European Union's Copernicus program, which has the operational support of the European Space Agency (ESA). Copernicus Sentinel data is made available under a [full, free and open license](https://sentinel.esa.int/documents/247904/690755/Sentinel_Data_Legal_Notice). This license makes it possible for Google to integrate the data in their catalog of collections and expose it to the many GEE users.
+
+ESA applies SAR processing to generate Level-1 data from raw Sentinel-1 signal data in 2 formats: Single Look Complex (SLC) and Ground Range Detected (GRD). SLC is required for interferometric and partial polarimetric parameter extraction (it contains the phase information), whereas GRD is intensity ("detected") data only, sampled in ground range. Google collects the GRD data.
+
+The Level-1 GRD processing assumes an elliptoid Earth surface model, thus [Level-1 GRD needs to be processed to create the Level-2 geocoded, calibrated backscattering coefficients](https://developers.google.com/earth-engine/sentinel1#sentinel-1-preprocessing) which end up in the GEE COPERNICUS/S1_GRD_FLOAT collection.
+
+The S1_GRD_FLOAT collection, and its log-scaled S1_GRD computed equivalent, contains "analysis ready" images, as they have square pixel spacing, with pixel values as FLOAT32, and a projection which defaults to the one of the local UTM zone. Thus, they can be combined with other images, used in feature extractions and reductions, etc.
```

Comment:
...and UTM projection in the auto-determined zone.

Reply:
revised